### DISCUSSION TITLE
Images don't render after deploying application with vercel but render properly both in development and production locally

### DISCUSSION THREAD

**Discussion by Muhu-Muza** [vercel/next.js#47144] [Help]
### Summary

Images on my portfolio site render properly on my local machine, however after deploying iwth vercel the images dont seem to get rendered by the browser. I read some where about adjusting my next.config.js file and i added my domains to it. However the issue still persists. I'm loading these images with links to my google drive.

next.config.js 
`/** @type {import('next').NextConfig} */
const nextConfig = {
  experimental: {
    appDir: true,
  },
  images: {
    domains: [
              'drive.google.com', 
              'portfolio-muhu-muza.vercel.app',
             ],
  },
}

module.exports = nextConfig
`
I'm using the inbuilt Image component to render my images and the image sources from my google drive like so.

 src="https://drive.google.com/uc?export=view&id=[id]"

**Comment by greenteaisgreat** [comment]
Do you know if your images are .svg? If they are, you'll need to add a vercel.json file at the root of your project. 
This is the most barebones config file you can use to allow .svg images. 

```vercel.json
{
  "images": {
    "sizes": [640, 750, 828, 1080, 1200],
    "dangerouslyAllowSVG": true
  }
}
```

However, I'm having this issue as well and incorporating this configuration still didn't solve my issue. Hopefully it'll help you out. Let me know if you ever figured out a way to display .svg images, if you were using .svg images.

**Comment by Ahukko8** [comment]
I also am in the shoes as you and found this on GitHub .let me know if you figure it out as well! Will do the same from my end too

**Reply by Daniel-Shunom** [reply]
@Ahukko8  I had the same issue too. I was using a JPG image when I had the problems, and switching to a PNG version of the image took care of that problem. Weirdly, JPG and Jpeg images don't work in prod. It might be a silent optimization feature.

**Reply by shekhardase** [reply]
> @Ahukko8 I had the same issue too. I was using a JPG image when I had the problems, and switching to a PNG version of the image took care of that problem. Weirdly, JPG and Jpeg images don't work in prod. It might be a silent optimization feature.

me using png still not working 
simple portfolio website works fine on local host when deployed only images are blank rest is working fine
